srcdeps-maven-local-repository/src/test/java/org/l2x6/srcdeps/localrepo/SrcdepsLocalRepositoryManagerTest.java

```diff
@@ -138,17 +170,17 @@ public void mvnGitProfileAndProperties() throws Exception {
         result.assertLogText("[echo] Hello [random name KMYTJDb9]!");
     }
 
-    @Test
+    // @Test
```

Comment:
Why is this commented out? Shouldn't you be using `@Ignore("This test is ignored because...")` ?